<p><b>[INFO]</b>&nbsp;POST Request : url=https://my-public-api-for-tests-production.up.railway.app/api/login <br> path_url=/api/login <br> headers={'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '42', 'Content-Type': 'application/json'} <br> body=b'{"username": "admin", "password": "12345"}' <br> </p><p><b>[INFO]</b>&nbsp;POST Response : url=https://my-public-api-for-tests-production.up.railway.app/api/login <br> status=502, reason=Bad Gateway <br> headers={'Content-Length': '120', 'Content-Type': 'application/json', 'Server': 'railway-edge', 'X-Railway-Fallback': 'true', 'X-Railway-Request-Id': '9akMlyJjRRCqdDmQeU1XEg_3165824431', 'Date': 'Sat, 15 Feb 2025 03:39:12 GMT'} <br> body={"status":"error","code":502,"message":"Application failed to respond","request_id":"9akMlyJjRRCqdDmQeU1XEg_3165824431"} <br> </p><p><b>[INFO]</b>&nbsp;/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host 'my-public-api-for-tests-production.up.railway.app'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings<br>  warnings.warn(</p><p><b>[INFO]</b>&nbsp;${response} = <Response [502]></p>